platform/qt/src/qmapboxgl.cpp

```diff
@@ -1062,6 +1314,15 @@ void QMapboxGL::setFilter(const QString& layer_, const QVariant& filter_)
     qWarning() << "Layer doesn't support filters";
 }
 
+/*!
+    Renders the map using OpenGL draw calls. If \a fbo is passed, it will
+    make sure to bind the framebuffer object before drawing, otherwise a
+    valid GL context is expected with an appropriated GL viewport state set
```

Comment:
In general, I don’t think “GL” is an appropriate abbreviation for “OpenGL” in public-facing documentation. Better to spell out “OpenGL”.